app/src/org/odk/collect/android/widgets/SelectOneAutoAdvanceWidget.java

```diff
@@ -90,7 +90,7 @@ public SelectOneAutoAdvanceWidget(Context context, FormEntryPrompt prompt) {
 
                 RadioButton r = new RadioButton(getContext());
                 r.setOnCheckedChangeListener(this);
-                r.setText(prompt.getSelectChoiceText(mItems.get(i)));
+                r.setText(stylize(prompt.getSelectChoiceText(mItems.get(i))));
```

Comment:
Not immediately clear how we should handle this, but i'd make sure we're defaulting these to not be markdowned.